recipes/ckanweb-efs-setup.rb

```diff
@@ -17,7 +17,7 @@
 
 # Update EFS Data directory for CKAN
 #
-include_recipe "datashades::httpd-efs-setup"
+include_recipe "datashades::efs-setup"
```

Comment:
does this not break things?

Reply:
what happened to using httpd-efs-setup.rb ?

Reply:
It's now on `ckanweb-setup.rb` line 28. So it will run on the web layer, but not batch.